geop/src/main/scala/com/azavea/usace/programanalysis/geop/GeopServiceActor.scala

```diff
@@ -2,19 +2,22 @@ package com.azavea.usace.programanalysis.geop
 
 import akka.actor.Actor
 
-import geotrellis.proj4.{ConusAlbers, LatLng}
-
 import org.apache.spark.SparkContext
 
 import scala.concurrent.future
 
-import spray.http.AllOrigins
+import spray.http.{ AllOrigins, MediaTypes }
 import spray.http.HttpHeaders.{`Access-Control-Allow-Headers`, `Access-Control-Allow-Methods`, `Access-Control-Allow-Origin`}
 import spray.http.HttpMethods.{DELETE, GET, OPTIONS, POST}
 import spray.json.{JsNumber, JsObject}
 import spray.routing.{Directive0, HttpService, RejectionHandler}
 
-import scala.collection.parallel.immutable.ParVector
+import geotrellis.raster.Tile
+import geotrellis.spark.io.{TileNotFoundError}
+import geotrellis.proj4.{LatLng, ConusAlbers}
+import geotrellis.raster.{IntConstantNoDataCellType}
```

Comment:
`import geotrellis.raster.Tile` can be combined with this line. Also, can we organize these alphabetically? Within the `geotrellis` group, I mean.